data_migration_script.py

```
@@ -0,0 +1,93 @@
+1. Backup your database first!
+2. python manage.py shell
+3. exec(open('data_migration_script.py').read())
+"""
+
+from master.models import (
+    CustomUser, JobProfile, JobImage, 
+    SegmentationType, Segmentation, Annotation, 
+    PolygonPoint, AnnotationIssue, ImageAnnotationIssue
+)
+
+def migrate_reviewer_data():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)